src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+
+public class Ui {
+    /**
+     * Print the greet expressions
+     */
+    public static void greet() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Hello! I'm Duke.");
+        System.out.println("    What can I do for you?");
+        System.out.println("    ____________________________________________________________");
+    }
+
+    public static void divider() {
+        System.out.println("    ____________________________________________________________");
+    }
+
+    /**
+     * Add task to taskList
```

Comment:
In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)